radio/src/gui/taranis/menu_general_setup.cpp

```diff
@@ -473,9 +477,9 @@ void menuGeneralSetup(uint8_t event)
         if (attr && s_editMode>0) {
           CHECK_INCDEC_GENVAR(event, g_eeGeneral.stickMode, 0, 3);
```

Comment:
The stick mode will change without pressing [Enter], I would prefer we change the reusable variable here